libs/generator/generator.h

```diff
@@ -0,0 +1,18 @@
+#ifndef ZKEMV_FRAMEWORK_LIBS_GENERATOR_GENERATOR_H_
```

Comment:
Probably I missed the discussion) OK, let's switch to `.hpp`, it is not a big deal. I used `.h` here because I had similar experience with the author of [this comment](https://stackoverflow.com/a/153301). LLVM, for example, also uses `.h`+`.cpp` files even though the headers can't be compiled with pure C compiler,

Reply:
Renamed